tests/python/unittest/test_target_codegen_vulkan.py

```diff
@@ -158,8 +158,54 @@ def build_f(f_ref):
     run_stress()
 
 
+@tvm.testing.requires_vulkan
```

Comment:
oops I didn't know that `test_target_codegen_vulkan.py` existed. We can merge `test_target_codegen_vulkan.py` and `test_target_codegen_spirv.py` if you like.

Reply:
Sound good, and I'd agree that it would be better to have them merged.  Commit incoming to merge both files into `test_target_codegen_vulkan.py`.